.github/workflows/build.yml

```diff
@@ -65,7 +67,7 @@ jobs:
         run: |
           mkdir build
           cd build
-          cmake ..
+          cmake .. -DLLAMA_ENABLE_WERROR
```

Comment:
The CI failed because cmake arguments need to be formatted like `-DLLAMA_ENABLE_WERROR=ON`.

Reply:
@cebtenzzre Thank you for pointing out the issue. That's what happens when you commit late at night, I'll more careful from now on. I've added the value to the arguments and even refactored the `LLAMA_ENABLE_WERROR` to `LLAMA_FATAL_WARNINGS`

Again, thank you for your feedback. If there's any more changes you would like, let me know 😄